### PR TITLE
Fix exception mappers

### PR DESCRIPTION
**Pull request by VladislavStefanov**

### CODE REVIEW

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.sap.scimono.api.API;
+import com.sap.scimono.entity.ErrorResponse;
+
+public class JsonParseExceptionMapper implements ExceptionMapper<JsonParseException> {
+
+  private static final String JSON_PARSE_EXCEPTION_MSG = "Parsing error in sent JSON.";
```
Comment (karaimin) [inline scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java]: Error parsing request body

Comment (karaimin) [inline scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java]: Error parsing request body

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonMappingExceptionMapper.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.sap.scimono.api.API;
+import com.sap.scimono.entity.ErrorResponse;
+
+
+public class JsonMappingExceptionMapper implements ExceptionMapper<JsonMappingException> {
```
Comment (karaimin) [inline scimono-server/src/main/java/com/sap/scimono/api/helper/JsonMappingExceptionMapper.java]: Check if it is possible to combine this exception mappers into one class

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+package com.sap.scimono.api.helper;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.sap.scimono.api.API;
+import com.sap.scimono.entity.ErrorResponse;
+
+public class JsonParseExceptionMapper implements ExceptionMapper<JsonParseException> {
```
Comment (karaimin) [inline scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java]: related to other similar class

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonProcessingExceptionMapper.java [hunk 4]
```diff
@@ -5,15 +5,18 @@
 import javax.ws.rs.ext.ExceptionMapper;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import com.sap.scimono.api.API;
 import com.sap.scimono.entity.ErrorResponse;
 
 public class JsonProcessingExceptionMapper implements ExceptionMapper<JsonProcessingException> {
 
+  private static final String JSON_PROCESSING_EXCEPTION_MSG = "One of the request inputs is not valid.";
```
Comment (karaimin) [inline scimono-server/src/main/java/com/sap/scimono/api/helper/JsonProcessingExceptionMapper.java]: same

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java (hunk 1 above)
Comment (VladislavStefanov) [reply scimono-server/src/main/java/com/sap/scimono/api/helper/JsonParseExceptionMapper.java]: Done

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonProcessingExceptionMapper.java (hunk 4 above)
Comment (VladislavStefanov) [reply scimono-server/src/main/java/com/sap/scimono/api/helper/JsonProcessingExceptionMapper.java]: Done

File: scimono-server/src/main/java/com/sap/scimono/api/helper/JsonMappingExceptionMapper.java (hunk 2 above)
Comment (VladislavStefanov) [reply scimono-server/src/main/java/com/sap/scimono/api/helper/JsonMappingExceptionMapper.java]: It is not possible to make one class implement both ExceptionMapper<JsonMappingException> and ExceptionMapper<JsonParseException>.
Can probably make one class implement ExceptionMapper<JsonProcessingException>

Comment (karaimin) [reply scimono-server/src/main/java/com/sap/scimono/api/helper/JsonMappingExceptionMapper.java]: No need to add multiple interfaces to achieve this behaviour. The current way is acceptable.

**Pull request merged**